framework/bsconfig.json

```diff
@@ -8,15 +8,12 @@
   "autoImportComponentScript": true,
   "createPackage": false,
   "stagingDir": "dist",
-  // "diagnosticFilters": [
-  //   {
-  //     "src": "**/roku_modules/**/*.*",
-  //     "codes": [
-  //       1107,
-  //       1009
-  //     ]
-  //   }
-  // ],
+  "diagnosticFilters": [{
+    "files": ["**/roku_modules/**/*.*"]
+  }, {
+    "files": ["**/ConsoleTestReporter.bs"],
```

Comment:
Why are we ignoring the console test reporter?

Reply:
Oh. Ok can you remove that exclusion as I'm about to go through and fix all the linting issues in general so this is going to go away and I don't want to miss it due to forgetting to removing these rules here.